libs/dataRequests.tsx

```diff
@@ -16,9 +16,11 @@ const getHeaders = () => {
 };
 
 const handleResponse = async (response: Response) => {
-  if (!response.ok) {
-    throw new Error(`Failed to fetch data. Status: ${response.status}`);
-  }
+  // TODO rework how we handle responses
+  // commented outt for the time being as was throwing errors on no subtopics
```

Comment:
when you get to this, look for a 404, that's our standard response on the api for empty thingy.